docs/docs-content/registries-and-packs/deploy-pack.md

```diff
@@ -271,238 +288,512 @@ Go ahead and review each of the following five files in the pack.
+Password:
+Login Succeeded
+```
+
+</TabItem> -->
+
+</Tabs>
+
+<br />
 
 ### Push the Pack to the Registry Server
-When you are logged in, push the pack to the registry server using the following command. 
+
+<Tabs groupId="registry-server">
+
+<TabItem label="Spectro Registry" value="Spectro_Registry">
+
 <br />
 
+Once you are logged in, push the pack to the registry server using the following command. 
+
+
 ```bash
 spectro pack push /packs/hello-universe-pack/
 ```
 
-You can verify that the pack is in the registry by using the `ls` command. This command lists all packs in the registry. 
-<br />
+To confirm that the pack is now in the registry, use the `ls` command. This command lists all packs available in the registry. 
+
 
 ```bash
 spectro pack ls
 ```
 
-Verify the pack you pushed is listed, as shown in the screenshot below.  
+Check if the pushed pack is listed, as shown in the image below.  
 
-<br />
 
 ![Screenshot of spectro pack ls](/tutorials/deploy-pack/registries-and-packs_deploy-pack_pack-push.png)
 
-<br />
 
-If you need help with the Spectro CLI commands, such as deleting a pack, refer to the [Spectro CLI commands](spectro-cli-reference.md#commands) guide. 
+For assistance with Spectro CLI commands, refer to the [Spectro CLI Commands](spectro-cli-reference.md#commands) guide. 
 <br /> 
 
-### Configure the Registry Server in Palette
-After you push the pack to the registry server, log in to Palette and configure the registry service so that you can access it when you create your cluster profile.   
+</TabItem>
 
+<TabItem label="ECR" value="ECR_Registry">
 
-Log in to [Palette](https://console.spectrocloud.com), and switch to the tenant admin view.
 <br />
 
-![Screenshot of Palette tenant settings.](/tutorials/deploy-pack/registries-and-packs_deploy-pack_tenant-admin.png)
+Once you are authenticated to your ECR registry, navigate to the **packs** directory, which contains the pack folder, **hello-universe-pack**.
+
+```bash
+cd /packs
+```
+
+Before pushing the pack to the registry, compress the contents of the pack folder into an archive file. Issue the command below to create the archive file.
+
+```bash
+tar -czvf $NAME-$VERSION.tar.gz hello-universe-pack
+```
+
+Now, proceed to push the **Hello Universe** pack to the ECR registry.
+
+```bash
+oras push $ACCOUNT_ID.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com/$REGISTRY_NAME/spectro-packs/archive/$NAME:$VERSION $NAME-$VERSION.tar.gz
+```
+
+You can use the `aws ecr describe-images` command to check if the pushed pack is listed in your ECR repository.
+
+```bash
+aws ecr describe-images --repository-name $REGISTRY_NAME/spectro-packs/archive/$NAME --region $AWS_DEFAULT_REGION
+```
+
+The image below displays the output of the `aws ecr describe-images` command, confirming the presence of the Hello Universe pack in the repository.
+
+![Screenshot of the output of the command aws ecr describe-images](/tutorials/deploy-pack/registries-and-packs_deploy-pack_pack-describe.png)
```

Comment:
We can use code ticks with this output instead of using a screenshot. You can highlight the three lines.  Checkout this [part](https://docusaurus.io/docs/markdown-features/code-blocks#highlighting-with-metadata-string) of the Docusarus docs.

Reply:
Done. What did you think? :)